src/tooltip/tooltip.js

```diff
@@ -276,6 +284,14 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
               ttScope.title = val;
             });
 
+            function watchNgDisabled() {
+              unbindWatcher = scope.$watch(attrs.ngDisabled, function (newValue, oldValue) {
```

Comment:
I stand corrected, `$observe` is a much better idea, more performant (less watchers on scope) and direct to the point.

Thanks for mentioning this @wesleycho , I finally went to dig up how attribute observers work. They're only called on `$set` so it's great. :smile: